RecoTracker/LSTCore/standalone/code/core/trkCore.h

```diff
@@ -8,6 +8,8 @@
 #include "ModuleConnectionMap.h"
 #include "SDLMath.h"
 #include "Event.h"
+#include <cppitertools/enumerate.hpp>
```

Comment:
https://github.com/SegmentLinking/TrackLooper/pull/376
I'm not sure we intended to go back

Reply:
For now I just copied the code from this branch https://github.com/SegmentLinking/TrackLooper/tree/lstod to get a working version. I can refactor the code later to get rid of cppitertools

Reply:
It would be good to do it now since the dependence is very minimal. It's only needed in two lines:

https://github.com/SegmentLinking/cmssw/blob/3c89f944593a5fc6447e1fa7007a445166ce7af8/RecoTracker/LSTCore/standalone/code/core/trkCore.cc#L451
https://github.com/SegmentLinking/cmssw/blob/3c89f944593a5fc6447e1fa7007a445166ce7af8/RecoTracker/LSTCore/standalone/code/core/trkCore.cc#L472

Reply:
I'd even suggest to rewrite the git history so that cppitertools doesn't show up in the logs/pack

Reply:
> I'd even suggest to rewrite the git history so that cppitertools doesn't show up in the logs/pack

The last update still has the original commit that adds `cppitertools`

Reply:
I've removed the commit with cppitertools